apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -145,8 +145,8 @@ function AssessmentQuestionRow({
                 href="#"
                 class="ms-2 text-info"
                 data-bs-toggle="tooltip"
-                data-boundary="window"
-                title="This question uses a rubric"
+                data-bs-boundary="window"
```

Comment:
TODO: make this change everywhere in another PR.

Reply:
Turns out we can't make this change yet.

```suggestion
                data-boundary="window"
```

TODO in another PR.